manuscript/manuscript.md

```diff
@@ -528,7 +536,7 @@ of interest as outlined in Rule 9.
 National Center for Supercomputing Applications.
 Software Carpentry Foundation.
 iDigBio/NSF.
```

Comment:
And I did find out that we should include some EPA verbiage.  So please include:

> Contributions from JWH, an employee of the U.S. Environmental Protection Agency (EPA)
> have not been subjected to the Agency's peer and administrative review. Therefore, the
> conclusions and opinions drawn are solely those of the author(s) and are not necessarily
> the views of the Agency.  Mention of trade names or commercial products does not
> constitute endorsement or recommendation for use.